robot/client/client.go

```diff
@@ -325,7 +325,7 @@ func (rc *RobotClient) connect(ctx context.Context) error {
 
 	client := pb.NewRobotServiceClient(conn)
 
-	refClient := grpcreflect.NewClient(rc.backgroundCtx, reflectpb.NewServerReflectionClient(conn))
+	refClient := grpcreflect.NewClientV1Alpha(rc.backgroundCtx, reflectpb.NewServerReflectionClient(conn))
```

Comment:
I changed this as the new version of `relectpb` outputs this lint warning:
```module/module.go:75:15: SA1019: grpcreflect.NewClient is deprecated: Use NewClientV1Alpha if you are intentionally pinning the v1alpha version of the reflection service. Otherwise, use NewClientAuto instead. (staticcheck)
        refClient := grpcreflect.NewClient(ctx, reflectpb.NewServerReflectionClient(conn))
                     ^
cli/client.go:593:15: SA1019: grpcreflect.NewClient is deprecated: Use NewClientV1Alpha if you are intentionally pinning the v1alpha version of the reflection service. Otherwise, use NewClientAuto instead. (staticcheck)
        refClient := grpcreflect.NewClient(refCtx, reflectpb.NewServerReflectionClient(conn))
                     ^
robot/client/client.go:328:15: SA1019: grpcreflect.NewClient is deprecated: Use NewClientV1Alpha if you are intentionally pinning the v1alpha version of the reflection service. Otherwise, use NewClientAuto instead. (staticcheck)
        refClient := grpcreflect.NewClient(rc.backgroundCtx, reflectpb.NewServerReflectionClient(conn))
                     ^
INFO File cache stats: 38 entries of total size 393.0KiB
INFO Memory: 36 samples, avg is 127.6MB, max is 443.1MB
INFO Execution took 3.499068041s
make: *** [lint-go] Error 1
```

I'm not sure if I should go to the new api version or pin to the existing behavior as, I am unfamiliar with this code so I did the latter to reduce risk of breaking things.

Reply:
I don't know what this means and would like to understand before LGTM'ing

Reply:
LGTM mod having someone who is familiar with this review this change, since it will be permanent

Reply:
@Otterverse could you please confirm whether this is the correct way to handle this deprecation warning?

Reply:
I would rather we move forward off the v1alpha version... But that needs some more work, beyond just these three fixes linter catches. (Ex: we're actually importing "google.golang.org/grpc/reflection/grpc_reflection_v1alpha" in places.)